tests/integration_tests_control_files/flow_solver/CMakeLists.txt

```diff
@@ -27,10 +27,10 @@ set(TEST_OUTPUT_DIR ${CMAKE_CURRENT_BINARY_DIR})
 #  WORKING_DIRECTORY ${TEST_OUTPUT_DIR}
 #)
 # ----------------------------------------
-#configure_file(viscous_taylor_green_vortex.prm viscous_taylor_green_vortex.prm COPYONLY)
-#add_test(
-#  NAME MPI_VISCOUS_TAYLOR_GREEN_VORTEX
-#  COMMAND mpirun -np ${MPIMAX} ${EXECUTABLE_OUTPUT_PATH}/PHiLiP_3D -i ${CMAKE_CURRENT_BINARY_DIR}/viscous_taylor_green_vortex.prm
-#  WORKING_DIRECTORY ${TEST_OUTPUT_DIR}
-#)
+configure_file(viscous_taylor_green_vortex.prm viscous_taylor_green_vortex.prm COPYONLY)
+add_test(
+  NAME MPI_VISCOUS_TAYLOR_GREEN_VORTEX
+  COMMAND mpirun -np ${MPIMAX} ${EXECUTABLE_OUTPUT_PATH}/PHiLiP_3D -i ${CMAKE_CURRENT_BINARY_DIR}/viscous_taylor_green_vortex.prm
+  WORKING_DIRECTORY ${TEST_OUTPUT_DIR}
+)
```

Comment:
comment this out since it currently doesnt check anything

Reply:
Fixed in commit